javatests/arcs/core/storage/handle/SingletonIntegrationTest.kt

```diff
@@ -180,4 +204,13 @@ class SingletonIntegrationTest {
                 mode = StorageMode.ReferenceMode
             )
     }
+
+    // TODO(mmandlis): make a testutil/ Time implementation and reuse in all tests.
+    // What package should it be in?
+    private class TimeImpl : Time() {
```

Comment:
@jasonwyatt I need your advice where is the right place to factor it out as a test helper class. thanks!

Reply:
it could probably go in `arcs.jvm.util.testutil.TimeImpl`, under a `testonly` `arcs_kt_library`

Reply:
this means that javatests/arcs/core/ tests will dep on a arcs/jvm package, is that ok?
(also it must be arcs_kt_jvm_library, because it uses jvm time functions)

Reply:
Right, it's fine for tests to dep on whatever they need to dep on. Also - all tests are JVM tests anyway.